### PR TITLE
Line 33 need to have scrollObject instead of param

### PR DESCRIPTION
**Pull request by praveen133t**
## Proposed changes

Line 33 needs to have scrollObject instead of param

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/writing-running-appium/ios-xctest-mobile-gestures.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ coordinates and duration.
 ```java
 // Java
 JavascriptExecutor js = (JavascriptExecutor) driver;
-Map<String, Object> params = new HashMap<>();
+Map<String, Object> scrollObject = new HashMap<>();
 scrollObject.put("direction", "down");
 scrollObject.put("element", ((RemoteWebElement) element).getId());
 js.executeScript("mobile: swipe", params);
```
Comment (imurchie) [inline docs/en/writing-running-appium/ios-xctest-mobile-gestures.md]: If the name is changed in line 33, it needs to be changed here, too. `params` no longer exists.

**Review by imurchie — CHANGES_REQUESTED** [review]
If it is clearer naming is `scrollObject`, then this is good. Either way it is the same thing, though.

**Review by praveen133t — COMMENTED** [review]
ok

**Comment by jlipps** [comment]
@praveen133t are you able to make the requested change?

**Comment by jlipps** [comment]
closing for now since there are conflicts.

**Pull request closed without merging**